tests/e2e.sh

```diff
@@ -28,6 +28,8 @@ metadata:
 items: []
 EOF
 )
+HELM_ERROR_SNIPPET="Helm template command results in error"
```

Comment:
why can't we have this function in the catalog repo to replace the existing function and e2e tests?

Reply:
I made a PR to add this in the catalog repo but running kpt:docker-build fails right now. Will keep functions here while I debug it.